heartbeat/processor/peerAuthenticationRequestsProcessor_test.go

```diff
@@ -271,6 +271,7 @@ func TestPeerAuthenticationRequestsProcessor_requestKeysChunks(t *testing.T) {
 	processor, err := NewPeerAuthenticationRequestsProcessor(args)
 	assert.Nil(t, err)
 	assert.False(t, check.IfNil(processor))
+	_ = processor.Close() // avoid data races
```

Comment:
does this component "suffer" from concurrent accesses problem?

Reply:
not really, deleted it